modules/gitops-logging-into-keycloak.adoc

```diff
@@ -0,0 +1,66 @@
+[id="logging-into-keycloak_{context}"]
+= Logging into Keycloak
+
+Log in to the Keycloak console after configuring the Argo CD instance to manage identities or roles and access provided to the various roles. To log in, first you have to get the Keycloak admin username and password.
```

Comment:
```suggestion
By logging in to the Keycloak console after configuring the Argo CD instance, you can manage identities or roles and define the permissions assigned to the various roles. To log in, 

.Prerequisites
* Obtain the Keycloak admin user name and password.
```
- The first sentence is awkward and maybe trying to say too much. Consider an edit similar to the one I suggest above.
- `user name` is two words unless referring to a command or a UI element